_data-prepper/pipelines/configuration/sources/pipeline.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
The `configuration` directory seems unnecessary. Just want to flag this for a future fix. This just makes the link longer (and shorter links are preferable for SEO). We can fix that by removing the `configuration` folder and adding redirects from the old link to the new link in all files under the `pipelines` directory. @vagimeli Could you please put in an issue to fix this?

Reply:
Issue https://github.com/opensearch-project/documentation-website/issues/8223